tiktorch/server/session/backend/supervisor.py

```diff
@@ -39,9 +38,7 @@ def has_work(self):
         return self._pipeline.max_num_iterations and self._pipeline.max_num_iterations > self._pipeline.iteration_count
 
     def forward(self, input_tensors):
```

Comment:
```suggestion
    def forward(self, input_tensors: Sample):
```

at least in my pyright setup this is not inferred.